plugins/outputs/execd/execd.go

```diff
@@ -81,7 +81,8 @@ func (e *Execd) Write(metrics []telegraf.Metric) error {
 	for _, m := range metrics {
 		b, err := e.serializer.Serialize(m)
 		if err != nil {
-			return fmt.Errorf("error serializing metrics: %s", err)
+			e.Log.Warn("Skipping metric due to a serialization error: %s", err)
```

Comment:
This decreases the log message severity from Error to Warning. Not sure if this is good.

Anyway, please use `%w` for errors.

Reply:
Using `%w` (i.e. error wrapping) does _not_ make sense for logging as we will never unwrap the error. It only makes sense when returning an error and allows us to test for the underlying (wrapped) error type.

Reply:
It is more clear what is expected as operand, it seems a good practice to always use `%w` when dealing with errors, just as you use `%d` for integers or `%t` for booleans.

Reply:
I believe I fixed it!